test/e2e/node/pod_resize.go

```diff
@@ -354,7 +354,7 @@ func doPodResizeSchedulerTests(f *framework.Framework) {
 		//     2. Delete pod2 + pod3 to make room for pod3.
```

Comment:
This sounds like a test that should be `[Serial]` ? What happens when another test pod lands on the same node?

----

`[Serial]` unfortunately won't run in e.g. `pull-kubernetes-e2e-kind` (because we're running tests in parallel and skipping serial), but it does in other places (e.g. conformance is run serially)

Someday maybe we'll rework that to run parallel then serial in one invocation, but it will be a messy change.

Reply:
It is run in serial: https://github.com/kubernetes/kubernetes/blob/609bf0020142256778818b3a9133bcf924d235bc/test/e2e/node/pod_resize.go#L420

I'm actually not sure why it flaked... maybe one of the other serial tests isn't cleaning up it's pods synchronously?